demo/netops-demo/templates/generate-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "generateName" . }}-config
```

Comment:
Need to add `labels` sections with Helm conventions (see other charts from stable)